docs/installation/upgrading.rst

```diff
@@ -6,26 +6,80 @@ Upgrading KSQL
 Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
 capacity to take over temporarily for unavailable, restarting servers.
 
+Upgrading to ksqlDB 5.5 from KSQL 5.4
+-------------------------------------
+
+#. Prepare a list of SQL statements for migration. One way is to dump the command
```

Comment:
5.4 -> 5.5 is not a rolling restart, ie you'll need to shut down all ksql instances and then start up all ksqldb instances, meaning there will be downtime - but please double check with ksql folks.

Reply:
can confirm